rules/preprocessing.smk

```diff
@@ -0,0 +1,65 @@
+import os
```

Comment:
Is it common practice to include imports above the documentation?

It just seem a bit odd having the imports first

Reply:
Weird that `black` didn't capture this. However, those import were no longer necessary therefore it has been removed!

Fixed: 21305ee51b64193213e31d112de1f6e5e17eff03